cleanlab/classification.py

```diff
@@ -206,14 +206,12 @@ class CleanLearning(BaseEstimator):  # Inherits sklearn classifier
       perfectly labeled (you are certain that there are no errors in that class).
 
     find_label_issues_kwargs : dict, optional
-      Keyword arguments to pass into :py:func:`filter.find_label_issues
-      <cleanlab.filter.find_label_issues>`. Particularly useful options include:
+      Keyword arguments to pass into `~cleanlab.filter.find_label_issues`. Particularly useful options include:
       `filter_by`, `frac_noise`, `min_examples_per_class` (which all impact ML accuracy),
       `n_jobs` (set this to 1 to disable multi-processing if it's causing issues).
 
     label_quality_scores_kwargs : dict, optional
-      Keyword arguments to pass into :py:func:`rank.get_label_quality_scores
-      <cleanlab.rank.get_label_quality_scores>`. Options include: `method`, `adjust_pred_probs`.
+      Keyword arguments to pass into `~cleanlab.rank.get_label_quality_scores`. Options include: `method`, `adjust_pred_probs`.
```

Comment:
see comment here:  https://github.com/cleanlab/cleanlab/pull/790#issuecomment-1666176986

We want the module name to be visible in the rendered documentation so that it is clearer this method lives in a different module